samples/kafka/quickstart/Node/README.md

```diff
@@ -0,0 +1,92 @@
+# Send and Receive Messages in Node using Azure Event Hubs for Kafka Ecosystem
+
+Azure Event Hubs is a highly scalable data streaming platform and event ingestion service, capable of receiving and processing millions of events per second. Event Hubs can process and store events, data, or telemetry produced by distributed software and devices. Data sent to an event hub can be transformed and stored using any real-time analytics provider or batching/storage adapters.
+
+An Azure Event Hubs Kafka endpoint enables users to connect to Azure Event Hubs using the Kafka protocol (i.e. Kafka clients). By making minimal changes to a Kafka application, users will be able to connect to Azure Event Hubs and reap the benefits of the Azure ecosystem. Kafka enabled Event Hubs currently supports Kafka versions 1.0 and later.
+
+This quickstart will show how to create and connect to an Event Hubs Kafka endpoint using an example producer and consumer written in Go.
+
+NOTE: This sample uses the [node-rdkafka](https://github.com/Blizzard/node-rdkafka) library which currently doesn't support Windows as well as Linux/Mac. For instructions on how to configure for Windows, please visit the node-rdkafka project and follow their instructions before continuing with the sample (though our sample is untested on Windows). However, this sample has been tested on the [Linux Subsystem on Windows 10](https://docs.microsoft.com/en-us/windows/wsl/install-win10), so that is a good option if you plan on running on Windows.
```

Comment:
I think this might be a typo?

> which currently **doesn't** support Windows as well as Linux/Max

Reply:
I think it's supposed to mean that it does support Windows, but not as well as it supports Mac/Linux. I will rephrase!